### PR TITLE
DOC: update docstring for series.nunique

### PR DESCRIPTION
**Pull request by cecilialiao**
Part of europandas sprint - expanded docstring for series.nunique @datapythonista

### CODE REVIEW

File: pandas/core/base.py [hunk 1]
```diff
@@ -1323,12 +1323,31 @@ def nunique(self, dropna=True):
...
+        int
+
+        See Also
+        --------
+        DataFrame.nunique: Method nunique for DataFrame.
+        Series.count: Count non-NA/null observations in the Series.
+
+        Examples
+        --------
+        >>> s = pd.Series([1,3,5,7,7])
```
Comment (datapythonista) [inline pandas/core/base.py]: missing spaces after commas (PEP-8)

**Review by datapythonista — COMMENTED** [review]
Looks great, just one minor thing.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @cecilialiao

**Comment by jreback** [comment]
thanks @cecilialiao

**Comment by cecilialiao** [comment]
Thanks for accepting my first contribution!

**Pull request merged**